src/pages/RecommendedProduct.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+// import { FaHeart, FaStar } from 'react-icons/fa';
+import { useAppDispatch, useAppSelector } from '../redux/hooks/hooks';
+import { getRecommendedProducts } from '../redux/features/recommendedProductSlice';
+// import ProductPageAddToCart from '../components/carts/ProductPageAddToCart';
+// import Button from '../components/buttons/Button';
+import { DynamicData } from '../@types/DynamicData';
+import RecommendedDesign from '../components/Recommend';
+
+interface RecommendedProps {
+	prodId: string;
+}
+
+const RecommendedProduct = ({ prodId }: RecommendedProps) => {
+	const dispatch = useAppDispatch();
+	const { products, isLoading, error } = useAppSelector(
+		(state) => state.recommendedProducts,
+	);
+
+	useEffect(() => {
+		if (prodId) {
+			dispatch(getRecommendedProducts(prodId)).unwrap();
+		}
+	}, [dispatch, prodId]);
+
+	return (
+		<>
+			<div className="text-2xl pb-2 mt-5">
+				<h2>You may also like:</h2>
+			</div>
+			<div className="grid mobile:grid-cols-2 gap-5 pb-4 tablet:grid-cols-3 ipad:grid-cols-3 laptop:grid-cols-4 desktop:grid-cols-4 h-full rounded-md">
+				{isLoading && <p>Loading...</p>}
```

Comment:
@JacquelineTuyisenge Find a best loader and replace this kind of loading

Reply:
done